fuse_core/test/launch_tests/test_parameters.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import unittest
+
+from ament_index_python.packages import get_package_prefix
+
+from launch import LaunchContext, LaunchDescription, LaunchService
+from launch.actions import EmitEvent, ExecuteProcess
+from launch.events import Shutdown
+
+import launch_pytest
+from launch_pytest.actions import ReadyToTest
+from launch_pytest.tools import process as process_tools
+
+from launch_ros.substitutions import FindPackageShare
+
+import pytest
+
+@pytest.fixture
+def test_proc():
+    test_root = os.path.join(
+        get_package_prefix('fuse_core'), '..', '..', 'build', 'fuse_core', 'test'
```

Comment:
It looks like this assumes the test is being run after being built with colcon, with the default name for the build directory. It also assumes the software has been installed. I would recommend not using `get_package_prefix()` at all. There are at least a couple good alternatives.

One alternative would be to set the [`WORKING_DIRECTORY` argument to `ament_add_pytest_test`](https://github.com/locusrobotics/fuse/pull/286/files#diff-8f0dbf48cc6fdbc4ec0aa1ed045b1a616e2d5d337bffadb59d75c2901173e74eR58) to `${CMAKE_CURRENT_SOURCE_DIR}` and use a relative path.

Another alternative would be to use [the `ENV` arugment to pass in the path to the parameters file](https://github.com/ament/ament_cmake/blob/d38014a000241c145904c4a16851cb5033d0286f/ament_cmake_pytest/cmake/ament_add_pytest_test.cmake#L41) as an environment variable and use that as the path here.